src/test/bundler_test.ts

```diff
@@ -94,13 +94,13 @@ suite('Bundler', () => {
     });
   });
 
-  test.skip('Handle <base> tag', () => {
+  test('Handle <base> tag', () => {
     const span = preds.AND(
         preds.hasTagName('span'), preds.hasAttrValue('href', 'imports/hello'));
     const a = preds.AND(
         preds.hasTagName('a'),
         preds.hasAttrValue('href', 'imports/sub-base/sub-base.html'));
-    return bundle('html/base.html').then((doc) => {
+    return bundle('test/html/base.html').then((doc) => {
```

Comment:
I converted all 41 tests in `bundler_test.ts` to `async await`.  I assume these were all `.then` because ported over from pre-typescript vulcanize.